build-aux/m4/bitcoin_qt.m4

```diff
@@ -99,6 +97,12 @@ AC_DEFUN([BITCOIN_QT_CONFIGURE],[
     BITCOIN_QT_CHECK([_BITCOIN_QT_FIND_LIBS_WITHOUT_PKGCONFIG])
   fi
 
+  BITCOIN_QT_CHECK([
+    if test "x$bitcoin_cv_qt551" != xyes; then
+      BITCOIN_QT_FAIL([Qt version not supported])
```

Comment:
configure shouldn't fail unless --with-gui is explicitly specified. Otherwise, it should just turn off GUI support...

Reply:
Yes, this is exactly the behavior. BITCOIN_QT_FAIL fails only if GUI has been requested. If not, it generates a warning. On a machine with Qt version 5.3.2 I get:

```
configure                 -> WARNING: Qt version not supported; bitcoin-qt frontend will not be built
configure --with-gui      -> error: Qt version not supported
configure --without-gui   -> no errors nor warnings
```